### DISCUSSION TITLE
Filtering feature

### DISCUSSION THREAD

**Discussion by mattseaboard** [generic-ui/generic-ui#41] [General]
Hi there. I read on [this ](https://generic-ui.com/guide/filtering) page that you are working on adding a filtering feature.  Do you have any sort of roadmap on when this feature will be implemented?

I looked for a roadmap for generic-ui grid but couldn't find one.  We're evaluating which grid we're going to go with and am just curious if there is an expected timeline for this feature, thanks!

**Comment by generic-ui** [comment]
Hi @mattseaboard,

Currently, we are finishing 0.16.0 release which will upgrade the view engine and migrate the grid to the newest version of angular. Simultaneously we are working on features required by the users, so please add your feature request to the https://github.com/generic-ui/generic-ui/issues page. Regarding filtering, it is almost finished, we put it on hold so we can work on other staff, but we will gladly get back to it. Thanks for providing feedback.

**Comment by vvgrad** [comment]
Hi @mattseaboard,

Is the project abandoned?
any news?

**Reply by mattseaboard** [reply]
No clue, it's not my project and there have been no updates from generic-ui.  You probably need to ask them, not me.

**Reply by vvgrad** [reply]
I'm sorry. I wrote your username by mistake ) 
Of course I wanted to ask @generic-ui

**Comment by pakkle** [comment]
They still maintain https://generic-ui.com/home

From: mattseaboard ***@***.***>
Date: Wednesday, October 26, 2022 at 9:22 AM
To: generic-ui/generic-ui ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [generic-ui/generic-ui] Filtering feature (Discussion #41)

No clue, it's not my project and there have been no updates from generic-ui. You probably need to ask them, not me.

**Comment by mattseaboard** [comment]
Regarding the filtering question, there is an open issue for it but sadly no progress or reply https://github.com/generic-ui/generic-ui/issues/42
